app/views/layouts/_header.html.erb

```diff
@@ -13,7 +13,7 @@
           <strong>(<%= @real_user.display_name %>)</strong>
         <% end %>
           <span id='user-id'>
-           <%= @current_user.display_name %> ⯆
+           <%= @current_user.display_name %> <%= "\u25be" %>
```

Comment:
Good catch! But can you use `\u25bc` instead (https://www.fileformat.info/info/unicode/char/25bc/index.htm), it's from the same Unicode version and is bigger, which I prefer.

Reply:
Sure, I avoided it initially because it's the same height as the font so it looks top heavy to me next to the other characters.